charter/charter.md

```diff
@@ -23,29 +23,27 @@ to carefully consider its impact on the diverse set of applications
 that have adopted QUIC as a transport.
 
 The second area of work is supporting the deployability of QUIC, which
-includes specifications and documents, such as it applicability and
+includes specifications and documents such as applicability and
 manageability statements, improved operation with load balancers, the
-specification of a logging format and schemas for QUIC and HTTP/3
-endpoints (qlog), etc.
+specification of a logging format, and more.
 
 The third area of work is the specification of new extensions to QUIC.
-Extensions intended for Standards Track need to have general
-applicability to multiple application protocols. The WG may also
-decide to publish extensions as Informational or Experimental
-documents, e.g., to allow vendors to publicly document deployed
-proprietary extensions or to enable wider experimentation with new
-protocol features.
+The WG will primarily focus on extensions to the QUIC transport layer,
+i.e., extensions to QUIC that have broad applicability to multiple
+application protocols. The WG may also allow publish specifications to
```

Comment:
Allow publish?

Reply:
Picky, picky :-) Yeah, the "allow" should not be there.

Reply:
```suggestion
application protocols. The WG may also publish specifications to
```